chains/options.go

```diff
@@ -205,6 +207,9 @@ func getLLMCallOptions(options ...ChainCallOption) []llms.CallOption { //nolint:
 	if opts.repetitionPenaltySet {
 		chainCallOption = append(chainCallOption, llms.WithRepetitionPenalty(opts.RepetitionPenalty))
 	}
+	if opts.streamingFuncSet {
```

Comment:
I'm not sure we need the bool for the `func` options; the default for these is `nil`, which should be ok?

So you could just append the option regardless of the bool?

Reply:
Updated.